File: FrontEnd/Self-Tracker/src/app/models/suggestion-dto.ts
export class SuggestionDTO {
    suggestionId: number;
    description: string;
    creationDate: string;

    constructor(suggestionId: number, description: string, creationDate: string) {
        this.suggestionId = suggestionId;
        this.description = description;
        this.creationDate = creationDate;
      }
  }
  

File: Backend/UserDataService/src/main/java/com/user/dto/SuggestionDTO.java
package com.user.dto;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SuggestionDTO {
	private Long suggestionId;
	private String description;
	private LocalDate creationDate;
}
